### DISCUSSION TITLE
List of Country Codes

### DISCUSSION THREAD

**Discussion by jxnxsdev** [tidal-music/discussions#28] [Q&A] [answered]
I'm currently trying to develop an npm package to access the api and want to put all the Country Codes in an Enum. Is there a list with all Country Codes somewhere so i can implement them?

**Comment by stefanmoro** [comment, marked as answer]
Hey @hdgamer1404Jonas , great to hear you're working on an npm package!

As you can see in the [developer portal docs](https://developer.tidal.com/documentation/api/api-overview), the country codes are given in [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) format. 

Hope that helps!

**Reply by jxnxsdev** [reply]
Thank you, that was what i was looking for
